website/integrations/services/nextcloud/index.mdx

```diff
@@ -63,245 +64,351 @@ authentik already provides some default _scopes_ with _claims_, such as:
+    - `authentik.company` is the FQDN of the authentik installation.
 
 :::note
-This documentation lists only the settings you need to change from their default values. Other changes might cause issues accessing your application.
+This documentation lists only the settings that you need to change from their default values. Be aware that any changes other than those explicitly mentioned in this guide could cause issues accessing your application.
 :::
 
-1. **Create an application in authentik:**  
-   Note the chosen slug as it will be used later.
+## Create an application and provider in authentik
+
+1. Log in to authentik as an administrator and open the authentik Admin interface.
+2. Navigate to **Applications** > **Applications** and click **Create with Provider** to create an application and provider pair. (Alternatively you can first create a provider separately, then create the application and connect it with the provider.)
 
-2. **Create a SAML provider:**  
-   In authentik, navigate to **Applications > Providers** and create a **SAML provider** with the following settings:
+    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
+        - Note the application slug because it will be required later.
+    - **Choose a Provider type**: select **SAML Provider** as the provider type.
+    - **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
+        - Set the **ACS URL** to `https://nextcloud.company/apps/user_saml/saml/acs`.
+        - Set the **Issuer** to `https://authentik.company`.
+        - Set the **Audience** to `https://nextcloud.company/apps/user_saml/saml/metadata`.
+        - Set the **Service Provider Binding** to `Post`.
+        - Under **Advanced protocol settings**, set an available signing certificate.
+    - **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.
 
-    - **ACS URL:**  
-      `https://nextcloud.company/apps/user_saml/saml/acs`
-    - **Issuer:**  
-      `https://authentik.company`
-    - **Service Provider Binding:**  
-      Post
-    - **Audience:**  
-      `https://nextcloud.company/apps/user_saml/saml/metadata`
-    - **Signing certificate:** Select any valid certificate.
-    - **Property mappings:** Select all managed mappings.
+3. Click **Submit** to save the new application and provider.
 
 :::note
-Depending on your Nextcloud configuration, you might need to use `https://nextcloud.company/index.php/` instead of `https://nextcloud.company/`.
+Depending on your Nextcloud configuration, you may need to use `https://nextcloud.company/index.php/` instead of `https://nextcloud.company/`.
```

Comment:
Actually, "might" is the correct term. There's a [remote] chance this "might" happen... whereas "may" can imply permissions to do something and also implies a greater chance of something happening...